C++/easy-problems/primechecker/primechecker-solution.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+
+int isPrime(int num) {
```

Comment:
C++ and C have a bool type. Use that instead of returning an integer value of 0 or 1.